docs/src/global_optimizers/global.md

```diff
@@ -51,7 +51,7 @@ Recommend `BBO()`.
 
 ## NLopt.jl
 
-NLopt.jl algorithms are chosen via `NLopt.Opt(:algname)`. Consult the
+NLopt.jl algorithms are chosen via `NLopt.Opt(:algname, nparameter)` or `NLO(:algname)` where `nparameter` is the number of parameters to be optimized . Consult the
```

Comment:
it's not parameters. Parameters of the optimization problem are not optimized.

Reply:
.

Reply:
Yes, but those aren't parameters. Those are the states of the optimization problem. An optimization problem also has parameters, but that's a separate thing.